scalingo/data_scalingo_notification_platform.go

```diff
@@ -0,0 +1,75 @@
+package scalingo
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/Scalingo/go-scalingo"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceScNotificationPlatform() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceScNotificationPlatformRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"display_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"logo_url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"available_event_ids": {
+				Type:     schema.TypeList,
```

Comment:
Isn't it supposed to be a `TypeSet`?
From https://www.terraform.io/docs/extend/schemas/schema-types.html#typelist:
> Used to represent an ordered collection of items

From https://www.terraform.io/docs/extend/schemas/schema-types.html#typeset
> TypeSet implements set behavior and is used to represent an unordered collection of items

Reply:
You're right, we don't ensure the order

Reply:
done